handbook/openid-connect/openid-connect.md

```diff
@@ -1,48 +1,71 @@
-There are three files that must be created to connect your SystemLink server to an OpenID Connect provider: `[provider-dns].conf`, `[provider-dns].client`, and `[provider-dns].provider`. The `[provider-dns]` portion of each filename must be the URL encoded fully qualified domain name. 
+6. Log into the SystemLink web application using OpenID connect. After logging in you should see the message **There are no available applications.** because there are no role mappings yet created for this user.
 
-> For example, a OpenID provider with the DNS `example.com:9999` would yield files named `oexample.com%3a9999.conf` , `example.com%3a9999.client`, and `example.com%3a9999.provider`. 
+7. Logout and log back into the SystemLink web application as the NI Web Server admin user. This is the user created during NI Web Server guided setup.
 
-These files do not exist for new SystemLink installations. Each file must be added to the `C:\Program Files\National Instruments\Shared\Web Server\conf\openidc` directory. The NI Web Server must be restarted for these changes changes to any of these files to take effect. 
+8. Go to **Security** > **Roles** and click the gear icon in the top right.
+
+9. Add an OpenID Claim mapping for the **Server Administrator** role.
+
+    !!! note ""
+        Details on mapping claims to roles is found in [**Mapping OpenID Connect Claims to SystemLink Workspaces and Roles**](#mapping-openid-connect-claims-to-systemlink-workspaces-and-roles).
+
+10. Log in as an OpenID connect user with a mapping for the **Server Administrator** role and confirm they have the correct privileges.
+
+11. To enable OpenID Connect as the only login option, go back to **NI Web Server Configuration** > **Authentication** and disable **Log in as users controlled by the web server**.
+
+<figure>
+  <img src="../oidc-webserver.png" width="500" />
+  <figcaption>Enable OpenID Connect in NI Web Server</figcaption>
+</figure>
+
+## OpenID Configuration Files in SystemLink Server
 
-SystemLink can be configured to support multiple OpenID Providers simultaneously by creating a `[provider-dns].conf`, `[provider-dns].client`, and `[provider-dns].provider` file for each provider.  It is required that the user ID used within SystemLink be unique across providers. This id takes the form `[sub_claim]@issuer` where this issuer is the DNS name of the OpenID Connect provider. You can see the id SystemLink uses for a user by viewing the details of that user in the SystemLink security application. 
+There are three files that must be created to connect your SystemLink server to an OpenID Connect provider: `[provider-dns].conf`, `[provider-dns].client`, and `[provider-dns].provider`. The `[provider-dns]` portion of each filename must be the URL encoded fully qualified domain name.
+
+!!! note ""
+    For example, a OpenID provider with the DNS `example.com:9999` would yield files named `example.com%3a9999.conf` , `example.com%3a9999.client`, and `example.com%3a9999.provider`.
```

Comment:
Should we consider using the example admonition instead of the text "For example"? As in:

```suggestion
!!! example
    An OpenID provider with the DNS `example.com:9999` would yield files named `example.com%3a9999.conf` , `example.com%3a9999.client`, and `example.com%3a9999.provider`.
```

![image](https://user-images.githubusercontent.com/7519484/98724711-c17a2080-2359-11eb-9337-a0ad283a305e.png)

That way it's easy to scan through and tell what's an example vs. a note. Unless I guess we're intending this to be a clarification rather than an example?

Reply:
Good suggestion. I went ahead and did this. I also updated the CSS for admonitions to support our theme as well.